### PR TITLE
LPD-39708 Add visual alerts in the start and end fields of calendar event creation

### PR DESCRIPTION
**Pull request by igor-franca**
Related [ticket](https://liferay.atlassian.net/browse/LPD-39708)

### CODE REVIEW

**Comment by igor-franca** [comment]
Hey @cgmonte, I will still validate the use of these alert messages with the product and design team. Because of this, I haven't created the language keys yet, but I would like to start reviewing the core of this solution.

**Comment by igor-franca** [comment]
Hey @cgmonte, I pushed a version of the language key that has not yet been officially validated. But as I didn't want to hold back on reviewing the test, I sent this force push.

Bearing in mind that the language keys used are being validated. 😄

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -19772,6 +19773,7 @@ this-field-provides-the-values-to-filter-elements=This field provides the values
 this-field-requires-a-valid-style-value=This field requires a valid style value. The value has been reset to the previously selected style.
 this-field-type-does-not-support-csv-export-file-format=This field type does not support the CSV export file format. Select JSON, JSONL, or JSONT.
 this-field-type-does-not-support-translations=This field type does not support translations.
+this-field-will-be-automatically-filled-if-the-field-is-empty-or-incomplete=This field will be automatically filled if the field is empty or incomplete.
```
[the remark this answers is not in the document]
Comment (cgmonte) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:19776]: som ai thinks that "This field will be automatically populated if it is empty or incomplete." is even better.

**Review by cgmonte — COMMENTED** [review]
Hey @igor-franca, it looks good to me. I made some comments that may lead to changes, but they are not change requests.

**Comment by igor-franca** [comment]
I will run the tests even without a definition in the language key, we will not have any harm to the test results by changing the value of the language key.

**Comment by igor-franca** [comment]
ci:test:calendar

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by igor-franca** [comment]
ci:report:57716993

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by igor-franca** [comment]
ci:forward

**Comment by igor-franca** [comment]
Hey @cgmonte, thanks for the review. Your improvement in the error message was used in the final solution!

**Pull request closed without merging**